Resources/config/mongodb.xml

```diff
@@ -208,13 +208,5 @@
         <service id="doctrine_mongodb.odm.symfony.fixtures.loader" class="Doctrine\Bundle\MongoDBBundle\Loader\SymfonyFixturesLoader" public="false">
             <argument type="service" id="service_container" />
         </service>
-
-        <service id="doctrine_mongodb.odm.entity_value_resolver" class="Symfony\Bridge\Doctrine\ArgumentResolver\EntityValueResolver">
```

Comment:
It looks like a breaking change if already someone is using `MapEntity` attribute with the ODM. Not a big deal but it must be documented in the UPGRADE file.

Reply:
hmmm not sure about the BC break, these services were added in `4.6.x` and never been released, so if someone is using it is because `EntityValueResolver` was registered as a service manually, in that case I guess it will continue working fine.

Reply:
> and never been released

We're fine changing logic given this argument :)